.gitignore

```diff
@@ -52,4 +52,5 @@ bootnode.key
 nodelogs/*
 nodelogs
 
-network.env
\ No newline at end of file
+network.env
+starter-script.sh
```

Comment:
What is `starter-script.sh`? If this is something specific to your workspace, remember you can set global and work-space scoped git configs & gitignores. So no need to add your files to each repo's gitignore

Reply:
Oh, and the script is also just the thing I put in the top-level conversation in the code block.